extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -667,30 +667,6 @@ static public function onArticleDelete( $article, &$user, &$reason, &$error ) {
 	}
 
 	/**
-	 * @param RecentChange $recentChange
-	 * @return bool
-	 */
-	static public function onRecentChangeSave( $recentChange ) {
```

Comment:
This refactors how wall notifications are generated when message is edited - now we directly generate notification from `EditBuilder`, so that we can pass the new `Revision` instance instead of having to query the database.

For Wall custom actions, `rc_this_oldid` is always 0, so this never worked.